CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 2.1.0-beta.1
```

Comment:
Suggestion: Group all `feat`s first and then list the `fix`es and last the `chore`s for easier readability.